src/platform-aws.c

```diff
@@ -308,6 +310,33 @@ static int configure_ep_inorder(struct fid_ep *ep, int optname, const char* optn
 	return 0;
 }
 
+/*
+ * For the RDMA protocol, try to set max msg size on the current endpoint
+ * to the size of the max message we send with fi_send. This allows the EFA
+ * provider to enable the zero-copy path.
+ *
+ * Returns 0 on success or -error code on unexpected failure.
+ */
+static int configure_ep_max_msg_size(struct fid_ep *ep)
+{
+	int ret = 0;
+
+	size_t eager_max_size = (size_t)ofi_nccl_eager_max_size();
+	size_t optval = NCCL_OFI_MAX(NCCL_OFI_MAX(sizeof(nccl_net_ofi_rdma_ctrl_msg_t), eager_max_size),
```

Comment:
for clarity, would prefer to see the two sizeofs in the same `max()` call, unclear to me if you mean to do min of max here.

Reply:
This is selecting the max size of messages we send with `fi_send`. We have 3 such messages:
1. Connection messages `nccl_ofi_rdma_connection_info_t` -> 236B
2. Ctrl messages -> 56B
3. Eager messages, that by default are 8KB, but can be changed by an env variable. 
So by default this would be 8KB, but, as an example, if the env variable is set to 0, this would be 236B.